circuits/cpp/barretenberg/ts/src/main.ts

```diff
@@ -85,7 +85,7 @@ export async function proveAndVerify(bytecodePath: string, witnessPath: string,
 
     debug(`verifying...`);
     const verified = await api.acirVerifyProof(acirComposer, proof, isRecursive);
-    console.log(`verified: ${verified}`);
+    process.stdout.write(`${verified}`);
```

Comment:
console.log is a proxy to process.stdout.write when ran in node -- I replaced all console.log statements since this is not expected to be ran in the browser and its clearer

Reply:
For un-smart people like me thinking "but we're using bb.js in the browser?!", this holds for `main.ts` as it defines the CLI.